CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(naoqi_driver)
 
+add_definitions(-std=gnu++11)
```

Comment:
I'm not sure why `set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} whatever)` is any better than `add_definitions`? and `target_compile_definitions(<target> PUBLIC -std=c++11)` actually adds `-D-std=c++11` which is pretty useless :-/
I'll update the PR with `set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} whatever)`